### ISSUE TITLE
WC REST API Multisite Issues

### ISSUE THREAD

**Issue by TheBBSAgency** [woocommerce/woocommerce#20894]
We've been working on a WooCommerce issue on a multisite site and found something that is throwing a ton of errors into the error log.  

We've been doing some testing with our server team and the Woo Subscriptions team and we believe we've narrowed the issue down to our site's calling of the WC REST API. When we were going through the error logs we found a ton of messages that look like this: 
[client 73.231.39.198:64492] WordPress database error Table 'wp_upmulti.wp_sorb_4_woocommerce_order_items' doesn't exist for query SELECT order_item_type, order_item_id, order_id, order_item_name FROM wp_sorb_4_woocommerce_order_items WHERE order_id = 97465 ORDER BY order_item_id; /* From [upswingpoker.com/wp-json/wc/v2/orders/network?_wpnonce=6d817f11c8&network_orders=true&blog_id=4] in [/nas/content/live/upmulti/wp-content/plugins/woocommerce/includes/data-stores/abstract-wc-order-data-store-cpt.php:331] */ made by require('wp-blog-header.php'), wp, WP->main, WP->parse_request, do_action_ref_array, WP_Hook->do_action, WP_Hook->apply_filters, call_user_func_array, rest_api_loaded, WP_REST_Server->serve_request, WP_REST_Server->dispatch, call_user_func, WC_REST_Network_Orders_Controller->network_orders, WC_REST_CRUD_Controller->get_items, WC_REST_Orders_Controller->prepare_object_for_response, WC_REST_Orders_Controller->get_formatted_item_data, WC_Order->get_data, WC_Abstract_Order->get_items, WC_Data_Store->__call, call_user_func_array, Abstract_WC_Order_Data_Store_CPT->read_items, referer: https://upswingpoker.com/wp-admin/

When the subscriptions support team dug deeper into they database they reported this: 
"Some part of your code is requesting the following URL: upswingpoker.com/wp-json/wc/v2/orders/network?_wpnonce=6d817f11c8&network_orders=true&blog_id=4 

Since this is a multisite, we aren't seeing the 404 from the REST API return, instead we're only seeing that the database is not accessible errors (we're unsure how WooCommerce handles multisites in these situations)."

Basically it appears that since it's a multi site something is querying EVERY site on the network (regardless if WooCommerce is active or not) and since it's not finding WooCommerce on site #4 on the network it's throwing this error.

There should be some kind of conditional logic that first checks to see if WooCommerce is active on a site before running.

**Comment by claudiosanches** [comment]
Please, do not duplicate issues, this is already reported in https://github.com/woocommerce/woocommerce/issues/20874

**Comment by TheBBSAgency** [comment]
Apologies. Forgot we posted it already.


 Vincent Wondra
President
The BBS Agency
920-786-8438
thebbsagency.com
vincent@thebbsagency.com


*In an effort to handle all support requests in a more expedient manner,
please send all support requests or requests for urgent assistance directly
to support@thebbsagency.com <support@thebbsagency.com>*

*If your matter requires my personal attention, I will reply to your email
as soon as I am able. If you require my immediate attention, please call
me.*

Confidentiality Notice: This e-mail message, including any attachments, is
for the sole use of the intended recipient(s) and may contain confidential and
privileged information. Any unauthorized review, copy, use, disclosure, or
distribution is prohibited. If you are not the intended recipient, please
contact the sender by reply e-mail and destroy all copies of the original
message.
